versions.props

```diff
@@ -9,7 +9,7 @@ com.netflix.feign:feign-*= 8.17.0
 com.palantir.remoting-api:* = 1.9.0
 com.palantir.safe-logging:* = 1.3.0
 com.palantir.tritium:tritium-registry = 0.8.0
-com.squareup.okhttp3:* = 3.9.0
+com.squareup.okhttp3:* = 3.11.0
```

Comment:
Now for Retry-After handling, let's discuss here. The behavior change is as follows:

1) 408 (CLIENT_TIMEOUT) is now automatically retried if Retry-After header is not set or is set to 0, as per above and here's the code: (https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/internal/http/RetryAndFollowUpInterceptor.java#L367). I don't think we particularly care about this, but in practice it would mean one extra retry: it will give up if THAT extra retry fails.
2) 503 are retried if Retry-After is set to 0, here's the code: https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/internal/http/RetryAndFollowUpInterceptor.java#L374. Again, I don't believe we ever set Retry-After in 503s and again it means one extra retry (it will bail if that retry fails). Someone could still do this by throwing javax.ws.rs.ServiceUnavailableException.